app/client/src/modules/ui-builder/ui/wds/WDSMultiSelectWidget/config/propertyPaneConfig/validations/defaultOptionValueValidation.ts

```
@@ -0,0 +1,111 @@
+  let message = { name: "", message: "" };
+
+  const DEFAULT_ERROR_MESSAGE = {
+    name: "TypeError",
+    message:
+      "value should match: Array<string | number> | Array<{label: string, value: string | number}>",
+  };
+
+  const hasLabelValue = (obj: LabelValue): obj is LabelValue => {
+    return (
+      _.isPlainObject(obj) &&
+      obj.hasOwnProperty("label") &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)